app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss

```diff
@@ -0,0 +1,182 @@
+// desktop authenticated account dropdown styling
+$padding-small-horizontal: 3px;
+
+.navigation-user_info-logged_in {
+  height: 100%;
+  margin-top: 7px;
+  margin-right: 15px;
+
+  &.open {
+    .user_info-logged_in-account_dropdown_trigger-icon {
+      transform: rotate(180deg);
+    }
+  }
+
+  @media (max-width: $screen-mobile-menu) {
+    display: none; // hides dropdown on mobile
+  }
+}
+
+.user_info-logged_in-account_dropdown_trigger {
```

Comment:
these are all navigation classes and should be namespaced as such, the logged_in bit is superfluous in my opinion

.navigation-user_info 
.navigation-user_info-dropdown_trigger
.navigation-user_info-user_name 
.navigation-user_info-avatar
etc